src/app/components/recommendations/recommendations.jade

```diff
@@ -1,9 +1,10 @@
 md-divider
 div(ng-cloak)
```

Comment:
nice, do not forget about adding styles
https://docs.angularjs.org/api/ng/directive/ngCloak